[2024-07-30T10:27:29.129+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:27:29.229+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-15T00:00:00+00:00 [queued]>
[2024-07-30T10:27:29.250+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-15T00:00:00+00:00 [queued]>
[2024-07-30T10:27:29.251+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:27:29.336+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-15 00:00:00+00:00
[2024-07-30T10:27:29.353+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1163) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:27:29.357+0000] {standard_task_runner.py:63} INFO - Started process 1168 to run task
[2024-07-30T10:27:29.360+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-15T00:00:00+00:00', '--job-id', '193', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpm2r0yjer']
[2024-07-30T10:27:29.366+0000] {standard_task_runner.py:91} INFO - Job 193: Subtask Run_data_quality_checks
[2024-07-30T10:27:29.536+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-15T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:27:30.047+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-15T00:00:00+00:00'
[2024-07-30T10:27:30.053+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:27:30.134+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:27:30.180+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:30.227+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:27:30.239+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:30.246+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 3360 records
[2024-07-30T10:27:30.251+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:27:30.310+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:30.390+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:27:30.400+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:30.413+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 81 records
[2024-07-30T10:27:30.416+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:27:30.454+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:30.508+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:27:30.680+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:30.688+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:27:30.690+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:27:30.740+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:30.785+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:27:30.888+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:30.903+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:27:30.905+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:27:30.961+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:31.050+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:27:31.061+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:31.068+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 3358 records
[2024-07-30T10:27:31.070+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:27:31.123+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-15T00:00:00+00:00, execution_date=20181115T000000, start_date=20240730T102729, end_date=20240730T102731
[2024-07-30T10:27:31.262+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:27:31.315+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
